docs/guides/upgrade-guide/manager.md

```diff
@@ -27,8 +28,8 @@ adjustments are necessary
        use a version smaller v2. The v2 of Gilt is not yet usable.
 
        ```
-       gilt overlay  # you have to do this 2x
-       gilt overlay
+       # for releases < 7.0.0 : gilt overlay && gilt overlay
+       make sync
```

Comment:
Have a look at https://github.com/osism/osism.github.io/blob/main/docs/guides/deploy-guide/services/ceph.mdx?plain=1#L69-L86. This makes it easier for the reader.

Reply:
Nice! Added it and made the legacy description a bit more verbose.